src/passes/TypeSSA.cpp

```diff
@@ -203,14 +345,24 @@ struct TypeSSA : public Pass {
     ReFinalize().runOnModuleCode(getPassRunner(), module);
   }
 
+  ExactTypes disallowedTypes;
+
+  void processExactTypes(const ExactTypes& exactTypes) {
```

Comment:
Perhaps `processDisallowedTypes`?

If so then `ExactTypes` could be `Types` or `TypeSet`.